src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AccountService.java

```diff
@@ -128,9 +134,30 @@ public Map<CreationEnum, List<?>> addUsers(List<PiUser> users, String issuerEmai
                 continue;
             }
 
+            Optional<MediaAndLegalApplication> application =
+                mediaLegalApplicationRepository.findByEmail(user.getEmail());
+
+            if (userRepository.findByEmail(user.getEmail()).isPresent()
+                && application.isPresent()) {
+                String emailSent = publicationService.sendNotificationEmailForDuplicateMediaAccount(user.getEmail(),
+                                                                            application.get().getFullName());
+                if (emailSent != null && !emailSent.isEmpty()) {
```

Comment:
this logic doesnt line up with what youve coded, this will never hit

Reply:
It is working, I tested it.

Reply:
got your point.